github-integration-its/src/main/java/org/jenkinsci/plugins/github_integration/SomeTest.java

```diff
@@ -0,0 +1,23 @@
+package org.jenkinsci.plugins.github_integration;
```

Comment:
I would like rename plugin classes near some 1.0 release to exclude logging overlap between plugins. PR code will keep as is, new under other package.